src/open_inwoner/openzaak/utils.py

```diff
@@ -52,6 +53,54 @@ def is_zaak_visible(zaak: Zaak) -> bool:
     return is_object_visible(zaak, config.zaak_max_confidentiality)
 
 
+def get_role_identification_display(rol: Rol) -> str:
```

Comment:
```suggestion
def get_role_name_display(rol: Rol) -> str:
```
You don't show here any id fields, so I think that `name` or `description` is better here